public_html/css/sass/game.scss

```diff
@@ -0,0 +1,76 @@
+$total-height: 100%;
+
+#game-view, #canvas-view, #color-palette-view, 
+.total-area {
+  height: $total-height;
+}
+
+#canvas-view, #color-palette-view {
+  background-color: white;
+  border: 1px solid black;
+}
+
+.word-panel {
+  height: 10%;
+  position: relative;
+  text-align: center;
+  background-color: #E0EBEB;
+
+  h2 {
+    margin: 0;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translateX(-50%) translateY(-50%);  
+  }
+}
+
+.working-area {
+  height: 90%;
+}
+
+.players-info {
+  width: 15%;
+  background-color: #75A3A3;
+}
+
+.canvas-area {
+  width: 60%;
+  background-color: #A3C2C2;
```

Comment:
Color literals like `#A3C2C2` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#A3C2C2` should be written as `#a3c2c2`